src/engines/L3/launch.js

```diff
@@ -78,8 +69,10 @@ You can remove the option from your \`src/index.js\`-file. And you can safely re
       renderer = null
     }
   }
-
-  renderer.init().then(shaderLoader).then(fontLoader).then(initApp)
+  shaderLoader()
+  fontLoader()
+  initApp()
+  // renderer.init().then(shaderLoader).then(fontLoader).then(initApp)
```

Comment:
there is no need to run these in sequence (anymore)? If so, then let's removed the commented out line here as well